packit_service/worker/handlers/abstract.py

```diff
@@ -209,6 +211,49 @@ def get_packit_commands_from_comment(
     return []
 
 
+class CeleryTask:
+    def __init__(self, task: Task):
+        self.task = task
+
+    @property
+    def retries(self):
+        return self.task.request.retries
+
+    def is_last_try(self) -> bool:
+        """
+        Returns True if the current celery task is run for the last try.
+        More info about retries can be found here:
+        https://celeryproject.readthedocs.io/en/latest/userguide/tasks.html#retrying
```

Comment:
The links are no longer up to date. The correct link is:

https://docs.celeryq.dev/en/latest/userguide/tasks.html#retrying

Same below